plugins/osx/osx.plugin.zsh

```diff
@@ -161,6 +161,15 @@ function itunes() {
 		vol)
 			opt="set sound volume to $1" #$1 Due to the shift
 			;;
+		playing)
+			local state=`osascript -e 'tell application "iTunes" to player state as string'`
+			if [ "$state" = "playing" ]; then
```

Comment:
Could use the zsh double-brackets here: `[[ $state == "playing" ]]`. It's preferred OMZ style; the single-bracket `[ ]` is more for portability.